src/status_im/transport/message/v1/protocol.cljs

```diff
@@ -50,34 +73,6 @@
                                         :topic   (transport.utils/get-topic constants/contact-discovery)}
                                        whisper-opts)}]}))
 
-(defn- prepare-recipients [public-keys db]
-  (map (fn [public-key]
-         (select-keys (get-in db [:transport/chats public-key]) [:topic :sym-key-id]))
-       public-keys))
-
-(fx/defn multi-send-by-pubkey
-  "Sends payload to multiple participants selected by `:public-keys` key. "
-  [{:keys [db] :as cofx} {:keys [payload public-keys success-event]}]
-  (let [{:keys [current-public-key web3]} db
-        recipients                        (prepare-recipients public-keys db)]
-    {:shh/multi-post {:web3          web3
-                      :success-event success-event
-                      :recipients    recipients
-                      :message       (merge {:sig     current-public-key
-                                             :payload payload}
-                                            whisper-opts)}}))
-
-;; TODO currently not used
-(defrecord Ack [message-ids]
-  message/StatusMessage
-  (send [this cofx chat-id])
-  (receive [this chat-id sig timestamp cofx]))
-
-(defrecord Seen [message-ids]
```

Comment:
seen messages are completely removed ?

Reply:
I think we don't use it, we always use `MessagesSeen`?